virtual_ship/expedition/checkpoint.py

```diff
@@ -0,0 +1,51 @@
+"""Checkpoint class."""
+
+from __future__ import annotations
+
+from pathlib import Path
+
+import yaml
+from pydantic import BaseModel
+
+from .instrument_type import InstrumentType
+from .schedule import Schedule
+
+
+class _YamlDumper(yaml.SafeDumper):
+    pass
+
+
+_YamlDumper.add_representer(
+    InstrumentType, lambda dumper, data: dumper.represent_data(data.value)
+)
+
+
+class Checkpoint(BaseModel):
+    """
+    Checkpoint is schedule simulation.
```

Comment:
I don't understand this comment; expand a bit what is meant?

Reply:
Was a typo. is this better?